quarkus-cli/src/test/resources/quarkus315dependencyUpdates.csv

```diff
@@ -0,0 +1,74 @@
+# https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.9#resteasy-reactive-extensions-renamed-to-quarkus-rest-gear-white_check_mark
+io.quarkus:quarkus-resteasy-reactive,io.quarkus:quarkus-rest
```

Comment:
I would say, that TSV (tab-separated) file would be easier to read, but I would not insist

Reply:
I tried using tabs as separators and IMHO it is not really much better to read. And using tabs would cause problems with editing the file, with true tabs vs spaces problem in IDEs etc.
I prefer to keep the commas here.